go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	k8s.io/client-go v0.24.2
 	k8s.io/utils v0.0.0-20220210201930-3a6ce19ff2f9
 	sigs.k8s.io/controller-runtime v0.12.3
-	sigs.k8s.io/gateway-api v0.5.1
+	sigs.k8s.io/gateway-api v0.5.1-0.20221110151456-3daef509bcc9
```

Comment:
I upgraded sigs.k8s.io/gateway-api go module to the main branch, which covers fixes of https://github.com/kubernetes-sigs/gateway-api/pull/1520.